docs/dev/uap/key_concepts.rst

```diff
@@ -0,0 +1,106 @@
+Key Concepts
+============
+
+This document is based on `preliminary discussions in our Google docs <https://docs.google.com/document/d/14H3Rc922-IYy6DIEV76_3hXCQPwl6OmKecJkAheTkI4/edit>`_.
+It describes the key concepts found in the core auth module, and how they touch on other key concepts from other
+modules.
+
+**Kolibri** is a platform with two major purposes:
+#. Serve educational content in "disconnected" scenarios, and support students and teachers with self-paced,
+peer-interactive, and blended learning practices.
+#. Collect information about users and their interaction with that content, and allow it to be synced to other devices
```

Comment:
There's the option of just not connecting to the internet, but what if in a classroom setting?